### PR TITLE
Enable SELinux on JeOS Tumbleweed

### PR DESCRIPTION
**Pull request by mgrossu**
- Related ticket: https://progress.opensuse.org/issues/160032
- Verification run:  https://openqa.opensuse.org/tests/4232872
Related MR: https://github.com/os-autoinst/opensuse-jobgroups/pull/470

### CODE REVIEW

File: schedule/security/selinux_jeos.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: selinux_jeos
+description: >
+    This is for SElinux test for JeOS Tumbleweed
```
Comment (ricardobranco777) [inline schedule/security/selinux_jeos.yaml:3]:
```suggestion
    This is a SElinux test for JeOS Tumbleweed
```

**Review by ricardobranco777 — APPROVED** [review]
Minor suggestion. LGTM.

File: schedule/security/selinux_jeos.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - security/selinux/print_se_context
+    - security/selinux/audit2allow
+    - security/selinux/semodule
+    - security/selinux/setsebool
+    - security/selinux/restorecon
+    - security/selinux/chcon
+    - security/selinux/chcat
+    - security/selinux/set_get_enforce
+    - security/selinux/selinuxexeccon
+conditional_schedule:
```
Comment (mimi1vx) [inline schedule/security/selinux_jeos.yaml:34]: why is here this conditional ? when both condition have same result ?

File: schedule/security/selinux_jeos.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    sle_os_tests:
+        DISTRI:
+            sle:
+                - security/selinux/selinux_smoke
+                - security/selinux/enforcing_mode_setup
+            opensuse:
+                - security/selinux/selinux_smoke
+                - security/selinux/enforcing_mode_setup
```
Comment (mloviska) [inline schedule/security/selinux_jeos.yaml:42]: Same as Ondrej mentioned, we dont really need the condition as the result is the same

**Review by mloviska — APPROVED** [review]
I would prefer to have in the main_common.pm as we would share the same schedule with SLE/Leap or whatever can come in the future. As of now and as it is an isolated case, let's keep yaml schedule

File: schedule/security/selinux_jeos.yaml (hunk 2 above)
Comment (mgrossu) [reply schedule/security/selinux_jeos.yaml]: Removed the conditional part.

File: schedule/security/selinux_jeos.yaml (hunk 3 above)
Comment (mgrossu) [reply schedule/security/selinux_jeos.yaml]: Removed the conditional part.

**Pull request merged**